### PR TITLE
VS Code Dev Container

### PR DESCRIPTION
**Pull request by juliekoubova**
Hey, I created a VS Code devcontainer configuration using ludeeus' integration development container.

### CODE REVIEW

File: .devcontainer/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+secrets.yaml
```
Comment (astrandb) [inline .devcontainer/.gitignore:1]: Add a secrets.yaml file with dummy data in .devcontainer/

File: .github/workflows/cron.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: Cron actions
```
Comment (astrandb) [inline .github/workflows/cron.yaml:1]: I pushed this file to this PR by mistake. It is present in the original dev container but i don't know if it is needed or not.

File: .github/workflows/pull.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+name: Pull actions
```
Comment (astrandb) [inline .github/workflows/pull.yml:1]: I pushed this file to this PR by mistake. It is present in the original dev container but i don't know if it is needed or not.

File: .github/workflows/push.yml [hunk 4]
```diff
@@ -0,0 +1,58 @@
+name: Push actions
```
Comment (astrandb) [inline .github/workflows/push.yml:1]: I pushed this file to this PR by mistake. It is present in the original dev container but i don't know if it is needed or not.

File: custom_components/miele/__init__.py [hunk 5]
```diff
@@ -29,17 +29,7 @@
 from . import config_flow
 from .api import AsyncConfigEntryAuth
 from .const import ACTIONS, API, DOMAIN
-from .devcap import (  # noqa: F401
```
Comment (astrandb) [inline custom_components/miele/__init__.py:32]: This change was pushed by me by mistake

**Review by astrandb — CHANGES_REQUESTED** [review]
I have not used this dev container before so don't be offended if my comments are irrelevant.
- The .vscode folder should be included. It contains necessary launch files etc.
- The github actions that are included in the original dev container are missing here. I don't know if it is good or bad.

**Comment by juliekoubova** [comment]
You’re right, forgot the .vscode files by mistake. Will add them!

**Comment by astrandb** [comment]
Thank you @juliekoubova  for the inspiration. A VS Code dev container is now set up.

**Pull request closed without merging**